### ISSUE TITLE
Missing matches with spanExpansion

### ISSUE THREAD

**Issue by Akron** [KorAP/Krill#39]
Given the following text
```
Eine besonders künstlerische Aufnahme entstand mit dem Blick durch den Berliner Rathausturm, auf den man den Zeppelin zufliegen sieht. Bei der vom Boden aufgenommenen Landung lässt sich die Mannschaft an Bord, die das Landungsseil in Richtung Boden abwirft, gut erkennen.
```
The following query
```
Rathausturm auf {1:[base=die]} []{0,10} {2:[base=die]}
```
should return the matches
```
1. Rathausturm, auf den man den
```
and
```
2. Rathausturm, auf den man den Zeppelin zufliegen sieht. Bei der
```
But currently only the first match is returned.
This is probably an issue with the SpanExpansion function.
(Reported by Bryan Jurish)

**Comment by Akron** [comment]
The text in question is ``WPD15/Z88/14646`` and it's openly available in our Wikipedia corpus.

**Comment by Akron** [comment]
Probably fixed in https://github.com/KorAP/Krill/commit/1bbd1be60ffd37dad436165947cd939ec2cd179c. To be evaluated.

**Comment by margaretha** [comment]
Fixed in f151c96f6376e5949363c4c931c109b887c87dd1.

**Comment by Akron** [comment]
Fixed in https://github.com/KorAP/Krill/commit/21e4ca26f039f7df6396d5ac6ab131497b3fe5fc .
